shark-graph/src/test/java/shark/HhpcTest.kt

```diff
@@ -0,0 +1,13 @@
+package shark
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import shark.internal.hppc.HHPC.mixPhi
+
+class HhpcTest {
+
+  @Test fun `11 and 14_723_950_898 has same hash`() {
```

Comment:
nit: `has` => `have` no?

Reply:
Might want to add a comment explaining why this is important?

Reply:
Consider moving this test next to the tests where those values are used to make it clear what this is about.